components/homme/src/share/control_mod.F90

```diff
@@ -608,4 +608,43 @@ subroutine test_timestep_make_parameters_consistent(par, nerr)
          write(iulog,'(a,i2)') 'test_timestep_make_parameters_consistent nerr', nerr
   end subroutine test_timestep_make_parameters_consistent
 
+
+!call thisroutine after timestep_make_... routines
+  subroutine control_params_init
+    use time_mod, only : nsplit, tstep, dt_remap, dt_tracer, dt_hv, dt_forcing, dt_run_sub 
+    use parallel_mod, only: abortmp
+#ifdef MODEL_THETA_L
```

Comment:
cound this code be moved into the model specific model_init_mod.F90?  that would let us get rid of the #ifdef MODEL_THETA_L stuff.  (I'm always trying to reduce #ifdef's )